.gitattributes

```diff
@@ -0,0 +1 @@
+*.ts text=auto eof=crlf
```

Comment:
Added this, so my git would use Windows line endings, because your linter demands that for some reason. 😄